### PR TITLE
Design changes and improvements

### PR DESCRIPTION
**Pull request by shirishgoyal**
Bootstrap UI code removal
MD based design changes
Status based color assignment to tasks

image

image

image

image

### CODE REVIEW

**Comment by dmorina** [comment]
layout is good, we should make the feed content centered, we can disable preview for now as we dont have more info to show; project card can be .75*width of that (too much white space between columns)
max number of colors if 2 to 3 (1 being primary, 2 secondary and third accent) statuses, errors are an exception; I would suggest not putting colors on table headers

**Comment by shirishgoyal** [comment]
Layout for task-feed is centered if no task is selected.

**Comment by dmorina** [comment]
ok, cool just disable selection then, it's not good ux if content changes position {width height change is ok}

**Comment by shirishgoyal** [comment]
For task selection, also thinking of doing it like below

image

**Comment by shirishgoyal** [comment]
Updated theme as per agreed palette 
- #FBF5AB (yellow)
- #DEE4EE (grey)
- #243058 (dark blue).

image

Added Animations using ngAnimate and ngFx

File: bower.json [hunk 1]
```diff
@@ -4,26 +4,27 @@
...
     "angular-oauth2": "3.0.1",
-    "angular-filter": "~0.5.4",
-    "angular-dragdrop": "1.0.9",
+    "angular-route": "1.4.0",
+    "angular-sanitize": "1.4.0",
+    "angular-smart-table": "2.0.2",
+    "angular-strap": "2.1.2",
     "angular-ui-sortable": "~0.13.4",
-    "ng-file-upload": "~5.0.9"
+    "bootstrap": "3.3.2",
```
Comment (dmorina) [inline bower.json]: you can drop this

File: bower.json [hunk 2]
```diff
@@ -4,26 +4,27 @@
...
-    "angular-strap": "2.1.2",
-    "ng-grid": "2.0.14",
-    "angular-smart-table": "2.0.2",
     "angular-material": "0.10.1",
     "angular-oauth2": "3.0.1",
-    "angular-filter": "~0.5.4",
-    "angular-dragdrop": "1.0.9",
+    "angular-route": "1.4.0",
+    "angular-sanitize": "1.4.0",
+    "angular-smart-table": "2.0.2",
```
Comment (dmorina) [inline bower.json]: this one too

File: bower.json [hunk 3]
```diff
@@ -4,26 +4,27 @@
...
+    "angular-strap": "2.1.2",
```
Comment (dmorina) [inline bower.json]: and this

**Comment by shirishgoyal** [comment]
Bootstrap packages removed.

**Comment by dmorina** [comment]
lgtm, (we will revisit design later)

**Comment by dmorina** [comment]
Marking this pr as complete, please don't add other things unless it's a bugfix, thanks

**Comment by shirishgoyal** [comment]
@dmorina I am ready with homepage as well. Should I add that to this PR itself or as a new one ?

**Comment by dmorina** [comment]
@shirishgoyal would be nice if you submit it as new PR

**Comment by aginzberg** [comment]
thanks for pushing this through shirish et al. under my account, it would be nice to have more space on the left before "welcome, .." otherwise lgtm for now and we'll definitely keep iterating :grinning:

**Comment by shirishgoyal** [comment]
Alignment fixed and hopefully it looks fine now.

image

**Comment by dmorina** [comment]
fix merge conflicts @shirishgoyal

**Comment by dmorina** [comment]
and please for the /task view give 66% to the left view and 33 to the right one, because the left view is the main one where the worker will submit his answers

**Comment by dmorina** [comment]
lgtm

File: bower.json [hunk 4]
```diff
@@ -4,26 +4,24 @@
...
     "angular-ui-sortable": "~0.13.4",
-    "ng-file-upload": "~5.0.9"
+    "bootstrap": "3.3.2",
+    "jquery": "2.1.4",
+    "json3": "~3.2.4",
+    "ng-file-upload": "~5.0.9",
+    "ng-grid": "2.0.14",
+    "ngFx": "~1.1.0",
+    "underscore": "1.8.3"
   },
```
Comment (nistala) [inline bower.json]: we haven't removed all references to smart table and angular strap.

**Comment by nistala** [comment]
platform depends on packages removed by this PR - angular strap and smart table.

**Pull request merged**